zeebe/broker/src/test/java/io/camunda/zeebe/broker/system/partitions/TestPartitionTransitionContext.java

```diff
@@ -83,6 +83,7 @@ public class TestPartitionTransitionContext implements PartitionTransitionContex
   private ControllableStreamClock clock;
   private SecurityConfiguration securityConfig;
   private MeterRegistry transitionMeterRegistry;
+  private String brokerVersion = PartitionTransitionContext.super.getBrokerVersion();
```

Comment:
💭 I considered for a moment whether just using `VersionUtil.getVersion()` would be better, but if (for whatever reason) the implementation were to change, then it would need to be adjusted here as well. And that's likely forgotten. So I guess this super usage makes sense here.